src/sql/src/session/vars.rs

```diff
@@ -3615,7 +3615,7 @@ where
     }
 
     fn visible(&self, user: &User, system_vars: Option<&SystemVars>) -> Result<(), VarError> {
-        if self.internal && user != &*SYSTEM_USER {
+        if self.internal && (user != &*SYSTEM_USER || user != &*SUPPORT_USER) {
```

Comment:
Not convinced this commit should be in this PR. Can it be moved to another?

Reply:
I moved this to #22810 along with some tests, waiting for @jkosh44 to approve.